Assets/Scripts/CSharp7/CSharp7.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class CSharp7 : MonoBehaviour
+{
+    [SerializeField] private int _skipCount;
+    [SerializeField] private int _takeCount;
+
+    private int _vectorCount = 5;
+    private List<int> xPositions;
```

Comment:
Don't use private fields unnecessarily.
In your case, you can pass a list to method parameters

Reply:
+?